convert/convert.go

```diff
@@ -169,37 +171,39 @@ func BedScoreToWigRange(infile string, reference map[string]*chromInfo.ChromInfo
 			current.Score = common.StringToInt(words[4])
 		}
 		chromIndex = getWigChromIndex(current.Chrom, wigSlice)
-		if wigSlice[chromIndex].Values[midpoint].Value != 0 {
+		if wigSlice[chromIndex].Values[midpoint] != 0 {
 			log.Fatalf("Multiple scores for one position.")
 		}
 		for k := current.ChromStart; k < current.ChromEnd; k++ {
 			//DEBUG: fmt.Printf("b[j].Chrom: %s, b[j].ChromStart: %d, b[j].ChromEnd: %d, k: %d, len(wigSlice[chromIndex].Values) %d\n", b[j].Chrom, b[j].ChromStart, b[j].ChromEnd, k, len(wigSlice[chromIndex].Values))
-			wigSlice[chromIndex].Values[k+1].Value = float64(current.Score)
+			wigSlice[chromIndex].Values[k+1] = float64(current.Score)
```

Comment:
Here we need to have current.Score be set, but on line 168/170 above we don't complain if they did not provide a score.  I think score probably needs to be mandatory in the input bed file.  I know this is not something introduced here, but something I am now noticing.

Reply:
Hmmm, I don't connect how the length of words being greater than or equal to 5 means that a score was not provided?

Reply:
The length of "words" is the number of columns in the bed file.  The score is the fifth field, so if the length is not at least 5, then the user did not supply a score for each bed region.  This should probably be an error because it doesn't make sense to convert the scores in the bed file to wiggle format if the bed file doesn't have a score in it.

Reply:
I'll do a separate pull request with all these convert.go changes, including this one.